yarn.lock

```diff
@@ -10924,7 +11057,7 @@ tslib@^1.9.0:
   integrity sha512-4krF8scpejhaOgqzBEcGM7yDIEfi0/8+8zDRZhNZZ2kjmHJ4hv3zCbQWxoJGz1iw5U0Jl0nma13xzHXcncMavQ==
 
 "ttn-lw@file:sdk/js":
-  version "3.0.0-pre"
+  version "3.0.0-rc1"
```

Comment:
is it related to this PR?

Reply:
Appears like it has been missed to update `yarn.lock` after we applied the `rc1` tag. I guess it's fair to do this here as well then.